stock_picking_batch_extended/wizard/stock_picking_to_batch_views.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="stock_picking_to_batch_form">
+        <field name="model">stock.picking.to.batch</field>
+        <field
+            name="inherit_id"
+            ref="stock_picking_batch.stock_picking_to_batch_form"
+        />
+        <field name="arch" type="xml">
+            <field name="batch_id" position="before">
+                <field name="name" attrs="{'invisible': [('mode', '=', 'existing')]}" />
+            </field>
+            <xpath expr="//field[@name='user_id']/.." position="after">
+                <label for="notes" attrs="{'invisible': [('mode', '=', 'existing')]}" />
+                <field
+                    name="notes"
+                    attrs="{'invisible': [('mode', '=', 'existing')]}"
+                />
+            </xpath>
+            <xpath expr="//field[@name='user_id']/../.." position="after">
+                <group attrs="{'invisible': [('mode', '=', 'existing')]}">
+                    <group>
+                        <field name="batch_by_group" />
+                    </group>
+                </group>
+                <group
+                    string="Create multiple batch picking grouped by..."
+                    attrs="{'invisible': ['|', ('batch_by_group', '=', False), ('mode', '=', 'existing')]}"
+                >
+                    <field name="group_field_ids" nolabel="1" colspan="2">
+                        <tree editable="bottom">
+                            <field name="sequence" widget="handle" />
+                            <field name="field_id" options="{'no_create': True}" />
+                        </tree>
+                    </field>
+                </group>
+            </xpath>
+            <xpath expr="//button[@name='attach_pickings']" position="after">
+                <button
+                    name="action_create_batch_with_mode_existing"
+                    string="Add to existing batch"
+                    attrs="{'invisible': [('mode', '=', 'create')]}"
+                />
+            </xpath>
```

Comment:
This is not having effect. What was the intention here?

Reply:
Sorry. The values that the "mode" field now has are "existing" or "new". In case the option to create a new grouping is chosen, the button to add the selected delivery orders to an existing grouping disappears. Now it functions correctly.